cmd/all-in-one/all_in_one_test.go

```diff
@@ -46,12 +47,13 @@ const (
 	agentURL      = "http://" + agentHostPort
 
 	getServicesURL         = queryURL + "/api/services"
-	getTraceURL            = queryURL + "/api/traces?service=jaeger-query&tag=jaeger-debug-id:debug"
 	getSamplingStrategyURL = agentURL + "/sampling?service=whatever"
 
 	getServicesAPIV3URL = queryURL + "/api/v3/services"
 )
 
+var getTraceURL = queryURL + "/api/traces/"
```

Comment:
Why is this defined here instead of in the `const` section above?

Reply:
We can't modify it if it is declared as `const`.

Reply:
Ah yes, you're right, I missed that.